docs: Rephrase Templates section (#5062)
coder/coder: docs/templates.md

### Edit templates

You can edit a template using the coder CLI. Only
[template admins and owners](./admin/users.md) can edit a template.

Using the CLI, login to Coder and run the following command to edit a single template:

```sh
coder templates edit <template-name> --description "This is my template"
```

Review editable template properties by running `coder templates edit -h`.

> Alternatively, you can pull down the template as a compressed file (`.tar.gz`) to your current directory:

Reviewer: I believe the original docs were wrong here as well, it's not gzipped AFAICT.

```suggestion
Alternatively, you can pull down the template as a tape archive (`.tar`) to your current directory:
```

```sh

> coder templates pull <template-name> file.tar.gz

Reviewer: ```suggestion
coder templates pull <template-name> file.tar
```

Author: Oh, you're right! It's `.tar` by default!

```

> Then, unzip it by running:

Reviewer: ```suggestion
Then, extract it by running:
```

```sh

> tar -xf file.tar.gz

Reviewer: ```suggestion
tar -xf file.tar
```
